gax/src/main/java/com/google/api/gax/rpc/BidiStreamingCallable.java

```diff
@@ -42,24 +42,91 @@
 @BetaApi("The surface for streaming is not stable yet and may change in the future.")
 public abstract class BidiStreamingCallable<RequestT, ResponseT> {
 
+  public interface ClientStreamCallBack<V> {
+    void call(ClientStream<V> stream);
+  }
+
   protected BidiStreamingCallable() {}
 
+  public abstract ClientStream<RequestT> call(
```

Comment:
Can this not be `protected`?

Reply:
No. If it was, `GrpcExceptionBidiStreamingCallable` won't be able to call it. I'm not sure which annotation we should use for this. Is it `@InternalApi`?